tests/integration/test_hyper_transformer.py

```diff
@@ -1846,3 +1851,80 @@ def test_with_tuple_returned_by_faker(self):
             ]
         })
         pd.testing.assert_frame_equal(result, expected_results)
+
+    methods_to_inputs = {
+        'update_transformers': {'column_name_to_transformer': {'C': UniformEncoder()}},
+        'update_transformers_by_sdtype': {'sdtype': 'boolean', 'transformer': UniformEncoder()},
+        'remove_transformers': {'column_names': ['C']},
+        'remove_transformers_by_sdtype': {'sdtype': 'boolean'},
+    }
+
+    @pytest.mark.parametrize('method_name', methods_to_inputs.keys())
+    def test_unvalid_multi_column(self, method_name):
+        """Test that the Hypertransformer handles invalid multi column transformers."""
+        # Setup
+        class BadDummyMultiColumnTransformer(DummyMultiColumnTransformerNumerical):
+
+            @classmethod
+            def _validate_sdtypes(cls, columns_to_sdtype):
+                raise TransformerInputError('Invalid sdtype')
+
+        dict_config = {
+            'sdtypes': {
+                'A': 'categorical',
+                'B': 'categorical',
+                'D': 'categorical',
+                'E': 'categorical',
+                'C': 'boolean',
+            },
+            'transformers': {
+                'A': UniformEncoder(),
+                ('B', 'D', 'C'): BadDummyMultiColumnTransformer(),
+                'E': UniformEncoder()
+            }
+        }
+
+        config = Config(dict_config)
+        ht = HyperTransformer()
+        ht.set_config(config)
+
+        # Run
+        expected_warning = re.escape(
+            "Transformer 'BadDummyMultiColumnTransformer' is incompatible with the "
+            "multi-column field '('B', 'D')'. Assigning default transformer to the columns."
+        )
+        with pytest.warns(UserWarning, match=expected_warning):
+            ht.__getattribute__(method_name)(**self.methods_to_inputs[method_name])
+
+        # Assert
+        new_config = ht.get_config()
+        expected_dict_config = {
+            'sdtypes': {
+                'A': 'categorical',
+                'B': 'categorical',
+                'D': 'categorical',
+                'E': 'categorical',
+                'C': 'boolean'
+            }
+        }
+        if method_name.startswith('update'):
+            expected_dict_config['transformers'] = {
+                'A': UniformEncoder(),
+                'E': UniformEncoder(),
+                'C': UniformEncoder(),
+                'B': UniformEncoder(),
+                'D': UniformEncoder()
+            }
```

Comment:
you could parametrize this part as well. Basically instead of method_to_inputs being a dict, make it a list of tuples where each tuple has the structure `(method_name, input, expected)`, and then just use those variables here accordingly

Reply:
That's a good point, done in [e6083f7](https://github.com/sdv-dev/RDT/pull/758/commits/e6083f79907c1491785a1063acb9ae4804f15293)